File: python/program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id="", nama="", stok="", harga="", jenis="", bahan="", warna="", untuk="", size="", merk=""):
        super().__init__(id, nama, stok, harga, jenis, bahan, warna)
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk
    
    # Getter dan Setter untuk 'untuk'
    def get_untuk(self):
        return self.__untuk
    
    def set_untuk(self, untuk):
        self.__untuk = untuk
    
    # Getter dan Setter untuk 'size'
    def get_size(self):
        return self.__size
    
    def set_size(self, size):
        self.__size = size
    
    # Getter dan Setter untuk 'merk'
    def get_merk(self):
        return self.__merk
    
    def set_merk(self, merk):
        self.__merk = merk

File: java/program/Baju.java
class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;
    
    Baju(){
    }
    
    public Baju(String untuk, String size, String merk){
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return untuk;
    }

    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }
}
